src/Configurations/Fields/Sections/Synchronization.js

```diff
@@ -8,29 +8,41 @@ import {
   Col,
   Row,
   KeyValue,
+  NoValue,
 } from '@folio/stripes/components';
 import {
   TextField,
   Select,
 } from '@folio/stripes-acq-components';
 
-import { TIME_UNITS } from '../../../const';
+const formatNumber = (value) => (
+  Math.sign(value) !== 1 ? 1 : value
+);
 
 export const Synchronization = ({ isNonInteractive }) => {
   const intl = useIntl();
   const { values } = useFormState();
 
+  const TIME_UNITS = [
+    { label: intl.formatMessage({ id: 'ui-remote-storage.minutes' }), value: 'minutes' },
+    { label: intl.formatMessage({ id: 'ui-remote-storage.hours' }), value: 'hours' },
+    { label: intl.formatMessage({ id: 'ui-remote-storage.days' }), value: 'days' },
+    { label: intl.formatMessage({ id: 'ui-remote-storage.weeks' }), value: 'weeks' },
+    { label: intl.formatMessage({ id: 'ui-remote-storage.months' }), value: 'months' },
+  ];
+
   return (
     <Accordion label={intl.formatMessage({ id: 'ui-remote-storage.synchronization.title' })}>
       <KeyValue label={intl.formatMessage({ id: 'ui-remote-storage.synchronization.schedule.title' })}>
         {isNonInteractive
-          ? intl.formatMessage(
-            { id: 'ui-remote-storage.synchronization.schedule.info' },
-            {
-              delay: values.accessionDelay,
-              unit: values.accessionTimeUnit,
-            },
-          )
+          ? values.accessionDelay
+            ? intl.formatMessage(
+              { id: `ui-remote-storage.synchronization.schedule.info.${values.accessionTimeUnit}` },
```

Comment:
This is a bit fragile IMHO.
What if by chance we get `values.accessionTimeUnit` which is not in the thanslations?